taxi/tests/test_models.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from taxi.models import Manufacturer, Car
+
+
+class ManufacturerModelTests(TestCase):
+
+    def test_manufacturer_str(self):
+        manufacturer = Manufacturer.objects.create(
+            name="Jaguar",
+            country="USD"
+        )
+        self.assertEqual(
+            str(manufacturer), f"{manufacturer.name} {manufacturer.country}"
+        )
+
+
+class DriverModelTests(TestCase):
+    #
+    def setUp(self):
+        self.username = "test"
+        self.password = "<PASSWORD>"
+        self.license_number = "1234567890"
```

Comment:
The `license_number` is set to 10 characters, but if the validation logic requires it to be 8 characters, this will cause a validation error. Ensure that the test data matches the model's validation requirements.